Makefile

```diff
@@ -27,11 +27,13 @@ env.d/kustomize:
 	cp env.d/kustomize.dist env.d/kustomize
 
 k8s-apply-config: ## Build and deploy the Kubernetes configuration for jitsi
-	@$(KUSTOMIZE) /data/k8s/overlays/$(JITSI_K8S_ENV) | bin/kubectl apply -f -
+	@$(KUSTOMIZE) /data/k8s/overlays/$(JITSI_K8S_ENV) > k8s/.k8s-built-config
+	bin/kubectl apply -f /data/k8s/.k8s-built-config
 .PHONY: k8s-apply-config
 
 k8s-build-config: ## Build and display the Kubernetes configuration for jitsi
-	@$(KUSTOMIZE) /data/k8s/overlays/$(JITSI_K8S_ENV)
+	@$(KUSTOMIZE) /data/k8s/overlays/$(JITSI_K8S_ENV) > k8s/.k8s-built-config
+	cat k8s/.k8s-built-config
```

Comment:
Why is the reason in this task to save the ouput in a file and then display this content file ? For the `apply` I understand your need and the problem you have on mac os. But the for the build, that only display what kustomize built, is it really needed to save the content in a file ?

Reply:
There is no direct need to store this in a file, it was designed this way to match with the `k8s-apply-config` behaviour. I agree that it is simpler just to print it to sdtout instead of saving it in a file.